spec/cypress/app_commands/scenarios/admin/school_participants.rb

```diff
@@ -1,16 +1,35 @@
 # frozen_string_literal: true
 
-school = FactoryBot.create(:school, name: "Test School", slug: "test-school")
+school = FactoryBot.create(:school, name: "Test School", slug: "test-school", urn: "123456")
 cohort = Cohort.find_or_create_by!(start_year: 2021)
 
-user_1 = FactoryBot.create(:user, :mentor, full_name: "Mentor User 1")
-user_1.mentor_profile.update!(school: school, cohort: cohort)
-user_2 = FactoryBot.create(:user, :mentor, full_name: "Mentor User 2")
-user_2.mentor_profile.update!(school: school, cohort: cohort)
-user_3 = FactoryBot.create(:user, :early_career_teacher, full_name: "ECT User 1", email: "young_prosacco@crist.net")
-user_3.early_career_teacher_profile.update!(school: school, cohort: cohort, mentor_profile: user_1.mentor_profile)
-user_4 = FactoryBot.create(:user, :early_career_teacher, full_name: "ECT User 2")
-user_4.early_career_teacher_profile.update!(school: school, cohort: cohort, mentor_profile: user_1.mentor_profile)
+FactoryBot.create(:participant_profile, :mentor,
+  user: FactoryBot.create(:user, full_name: "Mentor User 1"),
+  school: school,
+  cohort: cohort
+)
+FactoryBot.create(:participant_profile, :mentor,
+  user: FactoryBot.create(:user, full_name: "Mentor User 2"),
+  school: school,
+  cohort: cohort
+)
+FactoryBot.create(:participant_profile, :ect,
+  user: FactoryBot.create(:user, full_name: "ECT User 1"),
+  school: school,
+  cohort: cohort
+)
+FactoryBot.create(:participant_profile, :ect,
+  user: FactoryBot.create(:user, full_name: "ECT User 2"),
+  school: school,
+  cohort: cohort
+)
```

Comment:
I think those users used to be linked to a mentor, namely "Mentor User 1". Will this break something important in tests?

Reply:
AH, missed that! But it doesn't seem to affect tests, so I'd leave it as is?

Reply:
I guess it changes a visual test, it used to show a mentor of a user, now it shows "Not assigned", but it is not a major change.

Reply:
Just to be sure I've restored it to the original data :)